tests/materiallaws/test_notch_approximation_law.py

```diff
@@ -0,0 +1,565 @@
+            [481.382, 0.2486e-2, 452.873],
+            [484.974, 0.2508e-2, 455.570],
+            [488.566, 0.2531e-2, 458.251],
+            [492.159, 0.2553e-2, 460.915],
+            [495.751, 0.2576e-2, 463.563],
+            [499.344, 0.2599e-2, 466.195],
+            [502.936, 0.2621e-2, 468.811],
+            [506.528, 0.2644e-2, 471.411],
+            [510.121, 0.2668e-2, 473.994],
+            [513.713, 0.2691e-2, 476.561],
+            [517.306, 0.2714e-2, 479.113],
+            [520.898, 0.2738e-2, 481.648],
+            [524.490, 0.2761e-2, 484.168],
+            [528.083, 0.2785e-2, 486.672],
+            [531.675, 0.2808e-2, 489.160],
+            [535.268, 0.2832e-2, 491.632],
+            [538.860, 0.2856e-2, 494.089],
+            [542.452, 0.2880e-2, 496.531],
+            [546.045, 0.2905e-2, 498.957],
+            [549.637, 0.2929e-2, 501.368],
+            [553.230, 0.2953e-2, 503.763],
+            [556.822, 0.2978e-2, 506.144],
+            [560.414, 0.3002e-2, 508.510],
+            [564.007, 0.3027e-2, 510.860],
+            [567.599, 0.3052e-2, 513.196],
+            [571.192, 0.3077e-2, 515.518],
+            [574.784, 0.3102e-2, 517.824],
+            [578.376, 0.3127e-2, 520.117],
+            [581.969, 0.3153e-2, 522.395],
+            [585.561, 0.3178e-2, 524.658],
+            [589.154, 0.3203e-2, 526.908],
+            [592.746, 0.3229e-2, 529.144],
+            [596.338, 0.3255e-2, 531.365],
+            [599.931, 0.3281e-2, 533.573],
+            [603.523, 0.3307e-2, 535.768],
+            [607.116, 0.3333e-2, 537.948],
+            [610.708, 0.3359e-2, 540.116],
+            [614.300, 0.3385e-2, 542.270],
+            [617.893, 0.3412e-2, 544.411],
+            [621.485, 0.3438e-2, 546.539],
+            [625.078, 0.3465e-2, 548.654],
+            [628.670, 0.3492e-2, 550.756],
+            [632.262, 0.3519e-2, 552.845],
+            [635.855, 0.3546e-2, 554.922],
+            [639.447, 0.3573e-2, 556.986],
+            [643.040, 0.3600e-2, 559.038],
+            [646.632, 0.3627e-2, 561.078],
+            [650.224, 0.3655e-2, 563.106],
+            [653.817, 0.3682e-2, 565.122],
+            [657.409, 0.3710e-2, 567.126],
+            [661.002, 0.3738e-2, 569.118],
+            [664.594, 0.3766e-2, 571.098],
+            [668.186, 0.3794e-2, 573.067],
+            [671.779, 0.3822e-2, 575.025],
+            [675.371, 0.3850e-2, 576.971],
+            [678.964, 0.3878e-2, 578.906],
+            [682.556, 0.3907e-2, 580.830],
+            [686.148, 0.3935e-2, 582.744],
+            [689.741, 0.3964e-2, 584.646],
+            [693.333, 0.3993e-2, 586.537],
+            [696.926, 0.4022e-2, 588.418],
+            [700.518, 0.4051e-2, 590.289],
+            [704.110, 0.4080e-2, 592.149],
+            [707.703, 0.4109e-2, 593.999],
+            [711.295, 0.4138e-2, 595.839],
+            [714.888, 0.4168e-2, 597.668],
+            [718.480, 0.4198e-2, 599.488]
+        ])
```

Comment:
Consider putting datasets like this into a separate file to make the test more readable.

Like for example https://github.com/boschresearch/pylife/blob/develop/tests/materialdata/woehler/data.py